src/frontend/mame/clifront.cpp

```diff
@@ -422,8 +430,10 @@ void cli_frontend::listclones(const char *gamename)
 //  source file
 //-------------------------------------------------
 
-void cli_frontend::listbrothers(const char *gamename)
+void cli_frontend::listbrothers(const std::vector<std::string> &args)
 {
+	const char *gamename = args.size() > 0 ? args[0].c_str() : nullptr;
```

Comment:
Can you please change these to use `!args.empty()` instead of the comparison?  It's more readable and looks less dangerous without parentheses with used with the `?:` conditional operator.